### ISSUE TITLE
508/a11y: Evaluations page - complex image of process missing alt text 

### ISSUE THREAD

**Issue by iamjolly** [gsa-oes/office-of-evaluation-sciences#337]
The [Evaluations page](https://federalist-9dc074d2-9bb0-4720-8e37-036899ae1463.app.cloud.gov/preview/gsa-oes/office-of-evaluation-sciences/2021-website-updates-2.0/work/) features an image of the six steps OES uses in its process. There is no alternative text for the image (missing `alt` attribute entirely), and there is no adjacent text on the page that is equivalent to what is described visually in the image.

This fails [508/WCAG 2.1 - 1.1.1 Non-text Content - Level A](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content).

Short-term Recommendation:
Since what is shown in the image is a linear, six step process, consider using visibly-hidden text (using the `sr-only` utility class from USWDS) around an ordered list with text for each step in sequential list items. Use `alt=""` for the image since the steps in the image are fully explained in text for screen reader users.

Long-term Recommendation: 
Bring text from each step into the HTML of the page (instead of in the image) and use the icons for decorative purposes beside each step. This will reduce the amount of maintenance needed over time and ensure the content is accessible for the widest number of people. 

Screenshot:
<img width="862" alt="screenshot of process image missing alt text" src="https://user-images.githubusercontent.com/1093423/130104258-2c045169-5ba4-47a3-b6eb-f8916d767e55.png">
